build_web_compilers/pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: build_web_compilers
-version: 2.0.0-alpha.2
+version: 2.0.0
 description: Builder implementations wrapping Dart compilers.
 author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/build/tree/master/build_web_compilers
 
 environment:
-  sdk: ">=2.2.1-dev.4.0 <3.0.0"
+  sdk: ">=2.3.0-dev.0.1 <3.0.0"
```

Comment:
This will exclude flutter folks from getting this until they roll.

Are we against going with `2.3.0-dev.0.0`?

Reply:
We really need `2.3.0-dev.0.1` unfortunately

Reply:
ack